python/graphstorm/run/launch.py

```diff
@@ -323,6 +323,58 @@ def construct_torch_dist_launcher_cmd(
         master_port=master_port,
     )
 
+def wrap_dist_remap_command(
+        udf_command: str,
+        rank: int,
+        world_size: int,
+        with_shared_fs: bool,
+        num_trainers: int,
+        output_chunk_size: int = 100000,
+        preserve_input: bool = False):
+    """ Wrap distributed remap command
+
+        Parameters
+        ----------
+        udf_command:
+            Execution command.
+        rank:
+            Worker's rank in a distributed cluster.
+        world_size:
+            Total number of workers
+        with_shared_fs:
+            Whether all files are stored in a shared fs.
+        num_trainers:
+            Number of trainers on each machine.
+        output_chunk_size:
+            Number of rows per output file.
+        preserve_input:
+            Whether we preserve the input data.
+    """
+    # Get the python interpreter used right now.
+    # If we can not get it we go with the default `python3`
+    python_bin = sys.executable \
+        if sys.executable is not None and sys.executable != "" \
+        else "python3 "
+
+    udf_command[0] = "graphstorm.gconstruct.remap_result"
+
+    # Add remap related arguments
+    udf_command += ["--rank", str(rank)]
+    udf_command += ["--world-size", str(world_size)]
+    udf_command += ["--with-shared-fs", "True" if with_shared_fs else "False"]
+    udf_command += ["--num-processes", str(num_trainers)]
+    udf_command += ["--output-chunk-size", str(output_chunk_size)]
+    udf_command += ["--preserve-input", "True" if preserve_input else "False"]
```

Comment:
why do we give users this option?

Reply:
This is for debug purpose, in case the remap function has a bug.

Reply:
By default, it is False.